content/docs/use-cases/kubeflow-on-multinode-cluster.md

```diff
@@ -0,0 +1,251 @@
+
+```shell
+sudo apt install nfs-common
+sudo apt install nfs-kernel-server
+sudo mkdir /nfsroot
+```
+
+Than you need to configure /etc/exports to share that directory:
+
+```shell
+/nfsroot 192.168.0.0/16(rw,no_root_squash,no_subtree_check)
+```
+
+Notice that 192.168.0.0 is the nodes' CIDR, not the Kubernetes CIDR.
+
+### NFS Client
+
+Each node of the cluster must be able to establish connection to NFS server.
+This can be done by installing the right NFS client library in any node:
+
+```shell
+sudo apt install nfs-common
+```
+
+## Install Dynamic Provisioner
+
+Now you can create NFS PVs to enable each Pod to write its own data 
+in a common place from any node it will be spawned.
+ 
+In order to successfully complete the KubeFlow installation, a NFS PV must be created, to
+be binded to all the PVCs must be created.
+
+Since this operation can be tedious, Kubernetes has a mechanism to solve this problem
+and make creation of PVs based on existing PVCs automatic:
+[Dynamic Provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/).
+
+In next section we will install a Dynamic Provisioner for NFS volumes via Helm in our cluster.
+
+### Install Helm
+
+Helm is the Kubernetes packet manager. You can follow [this guide](https://helm.sh/docs/using_helm/#install-helm) to install it in a couple of steps.
+
+### Install NFS Provisioner
+
+The Dynamic Provisioner we are going to install will be able to create PVs for any PVC for a given [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/).
+
+A storage class is a label associated to volumes to specify a class of storage: using storage class 
+definition is possible to query and provision volumes with different performances or capabilities
+ (e.g SSD or slower disks)
+
+A special storage class is *default*: any PV or PVC that doesn't
+specify one is associated to it.
+
+NFS client provisioner can be found [here](https://github.com/helm/charts/tree/master/stable/nfs-client-provisioner).
+
+You can install it with Helm:
+
+```shell
+helm install 
+  --name nfs-client-provisioner \
+  --set nfs.server=<NFS Server IP> \
+  --set nfs.path=/exported/path \
+  --set storageClass.name=nfs \
+  --set storageClass.defaultClass=true \
+  stable/nfs-client-provisioner
+```
+
+The component in the system will add a *nfs* Storage Class that you can see in this way:
+
+```shell
+kubectl get storageclass -n kubeflow
+
+NAME                   PROVISIONER                            AGE
+nfs (default)          cluster.local/nfs-client-provisioner   6h13m
+...
+```
+
+As you noticed we specified an optional parameter storageClass.defaultClass to true: 
+this will define *nfs* storage class as default.
+In this way during KubeFlow installation, all PVCs will be labelled
+with *nfs* storage class.
+
+## Finally: install KubeFlow!
+
+We finally prepared our on-prem multi-node Kubernetes cluster to manage transparently volumes
```

Comment:
Replace this intro paragraph with:

`Now that you have prepared your on-prem, multi-node Kubernetes cluster to manage volumes using an NFS server, you can install Kubeflow using the [Kubeflow on Kubernetes](/docs/started/getting-started-k8s/) guide.`